tests/cases/clear_with_depth.amber

```diff
@@ -40,7 +40,7 @@ END
 # We would need a second pipeline or a depth test on the pipeline to observe the effects of the
 # depth-stencil clear.
 
-CLEAR_COLOR my_pipeline 10 20 30 40
+CLEAR_COLOR my_pipeline 255 0 0 255
```

Comment:
Why did this need to change?

Reply:
10 20 30 40 is not a color that we could verify by looking at it.

Reply:
True, this probably didn't have to change.

Reply:
ah.  :-)